starcheck/utils.py

```diff
@@ -282,9 +296,10 @@ def check_hot_pix(idxs, yags, zags, mags, types, t_ccd, date, dither_y, dither_z
              star or fid info to make a warning.
     """
 
-    eff_t_ccd = get_effective_t_ccd(t_ccd)
-
-    dark = aca_dark.get_dark_cal_image(date=date, t_ccd_ref=eff_t_ccd, aca_image=True)
+    dark_props = aca_dark.get_dark_cal_props(date=date, include_image=True,
+                                             aca_image=True)
+    dark = dark_props['image']
+    dark_t_ccd = dark_props['t_ccd']
```

Comment:
OK, looks like you are doing the next part of this project now.

Reply:
Well, since it was two weeks later and it wasn't going to get merged as-is anyway.